Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/Features/Tags/GET.feature

```diff
@@ -8,3 +8,8 @@ User requests available Tags
         Then the response status code is 200 (OK)
         And the response supports the English language
         And the response attributes contain tags
+
+    Scenario: Requesting the tags with the current hash
+        Given the most current hash h
+        When A GET request to the /Tags endpoint gets sent with hash h
+        Then the response status code is 304 (Not modified)
```

Comment:
I'd suggest a more generic phrasing (no need to leak the fact that the ETag is a hash; it might as well be a timestamp or an incrementing number)

Also, you should check that the response content is empty.

Given a list of tags l with an ETag e
And l didn't change since the last fetch
When a GET request to the /Tags endpoint gets sent with the If-None-Match header set to e
Then the response status code is 304 (Not modified)
And the response content is empty